README.md

```diff
@@ -107,6 +107,77 @@ To run test suit use 'check' target, e.g in build directory run:
 
     make check
 
+note that sio2jail may not support newer gcc / libc versions you may be using
+(#40). If so you can enable compiling test binaries in docker with cmake option:
+
+    -DTEST_COMPILER_IN_DOCKER=YES
+
+By default debian bullseye and gcc 10 will be used. You use other versions with
```

Comment:
```suggestion
By default debian bullseye and gcc 10 will be used. You can use other versions with
```

Reply:
As discussed on IRC we'll add compiling test programs in python instead of in docker